packages/aws-cdk-lib/aws-rds/lib/cluster-engine.ts

```diff
@@ -665,7 +697,19 @@ export class AuroraMysqlEngineVersion {
    *   defaults to "5.7"
    */
   public static of(auroraMysqlFullVersion: string, auroraMysqlMajorVersion?: string): AuroraMysqlEngineVersion {
-    return new AuroraMysqlEngineVersion(auroraMysqlFullVersion, auroraMysqlMajorVersion, auroraMysqlMajorVersion ? auroraMysqlMajorVersion !== '5.7' : false);
+    // Detects whether the auto-pause feature is supported.
+    // https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/aurora-serverless-v2-auto-pause.html#auto-pause-prereqs
+    const coercedVersion = semver.valid(semver.coerce(auroraMysqlMajorVersion));
+    const serverlessV2AutoPauseSupported = auroraMysqlMajorVersion === '8.0'
+      ? auroraMysqlFullVersion >= '8.0.mysql_aurora.3.08.0'
```

Comment:
Does this kind of comparison work reliably? Thinking about the comparison between `3.9.0` and `3.08.0` for example (not sure if this is realistic)